packages/Dispatch/readme.md

```diff
@@ -357,6 +359,21 @@ To fix this, make sure you are using the correct agent pool. In order to success
 
 ![azurepipelinesagentpoolvmimages](https://user-images.githubusercontent.com/41968495/52246146-8ea81c00-2899-11e9-8ed1-5a0347ad12a5.jpg)
 
+## <a name="faq"></a>Frequently Asked Questions
```

Comment:
Hi Tien, what tool are you using to author?  In typora, these don't render properly..